pdns/responsestats.cc

```diff
@@ -53,7 +53,7 @@ map<uint16_t, uint64_t> ResponseStats::getQTypeResponseCounts() const
 {
   map<uint16_t, uint64_t> ret;
   uint64_t count;
-  for (unsigned int i = 0; i < 65535; ++i) {
+  for (unsigned int i = 0; i < d_qtypecounters.size(); ++i) {
```

Comment:
Why not use the `auto& entry : d_qtypecounters` syntax here as well?

Reply:
hmm, we need the index